docs/UserGuide.md

```diff
@@ -240,7 +241,7 @@ Format: `listorder INDEX`<br>
 Examples:
 * `listcustomer` followed by `listorder 2` lists all orders of the 2nd customer in the address book.
 * `findcustomer Betsy` followed by `listo 1` lists all orders of the 1st customer from the list of customers found with "Betsy".
-
+  ![result for 'listo 1'](images/listOrder.png)
```

Comment:
this image does not match that of betsy order as well

Reply:
Updated in commit.